ITISFoundation/osparc-simcore
# ✨ Integrate locust dashboards

## What do these changes do?
- Integrate locust-plugins' [dashboards](https://github.com/SvenskaSpel/locust-plugins/tree/master/locust_plugins/dashboards) with our load testing framework (as demoed in the review meeting)
- These dashboard allow users to view (in real time) statistics based on our locust load tests.
- I also did a minor clean up of make recipes and unified settings which were still not handled by env vars

![image](https://github.com/ITISFoundation/osparc-simcore/assets/126242332/f30de128-8fdf-430f-915b-1b9773d49373)
![image](https://github.com/ITISFoundation/osparc-simcore/assets/126242332/71d06609-4e87-43e0-ab1e-3fbd0088bc8b)


## Related issue/s


- #5953 

## How to test



See [README](https://github.com/ITISFoundation/osparc-simcore/blob/ea8da8294615c7ecdcdc3436a94e2da7fcf52525/tests/performance/README.md) for how to run load tests and visualize

## Dev-ops checklist
